From c02a2b8ad3efb0b59392b1c2d441e3f7144c1fda Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@lantiq.com>
Date: Tue, 21 Jul 2015 22:45:40 +0200
Subject: [PATCH] csdk: remove dependency to liboc_logger.so in c code

The liboc_logger.so is only used by the c++ part and not by the c part,
remove it from the c only part.

Change-Id: Ib0894974de8ef7150401f02fbd0ea864d1d6f2cd
Signed-off-by: Hauke Mehrtens <hauke.mehrtens@lantiq.com>
Reviewed-on: https://gerrit.iotivity.org/gerrit/1801
Tested-by: jenkins-iotivity <jenkins-iotivity@opendaylight.org>
Reviewed-by: Charlie Lenahan <charlie.lenahan@intel.com>
Reviewed-by: Erich Keane <erich.keane@intel.com>
---
 resource/csdk/connectivity/test/SConscript         |    3 ---
 .../samples/linux/SimpleClientServer/SConscript    |    2 +-
 .../csdk/stack/samples/linux/secure/SConscript     |    2 +-
 3 files changed, 2 insertions(+), 5 deletions(-)

--- a/resource/csdk/connectivity/test/SConscript
+++ b/resource/csdk/connectivity/test/SConscript
@@ -58,9 +58,6 @@ target_os = env.get('TARGET_OS')
 if target_os not in ['arduino', 'darwin', 'ios']:
 	catest_env.AppendUnique(LIBS=['rt'])
 
-if target_os != 'darwin':
-    catest_env.PrependUnique(LIBS = ['oc_logger'])
-
 if env.get('SECURED') == '1':
     catest_env.AppendUnique(LIBS = ['tinydtls'])
 
--- a/resource/csdk/stack/samples/linux/SimpleClientServer/SConscript
+++ b/resource/csdk/stack/samples/linux/SimpleClientServer/SConscript
@@ -41,7 +41,7 @@ samples_env.AppendUnique(LIBPATH = [env.
 if target_os in ['darwin', 'ios']:
 	samples_env.PrependUnique(LIBS = ['m','octbstack', 'ocsrm', 'connectivity_abstraction','coap' ])
 elif target_os not in ['arduino']:
-	samples_env.PrependUnique(LIBS = ['m', 'octbstack', 'ocsrm', 'oc_logger', 'connectivity_abstraction', 'coap'])
+	samples_env.PrependUnique(LIBS = ['m', 'octbstack', 'ocsrm', 'connectivity_abstraction', 'coap'])
 	samples_env.AppendUnique(LIBS = ['rt'])
 
 if env.get('SECURED') == '1':
--- a/resource/csdk/stack/samples/linux/secure/SConscript
+++ b/resource/csdk/stack/samples/linux/secure/SConscript
@@ -41,7 +41,7 @@ if target_os not in ['windows', 'winrt']
 		samples_env.AppendUnique(LIBS = ['-lpthread'])
 
 samples_env.AppendUnique(LIBPATH = [env.get('BUILD_DIR')])
-samples_env.PrependUnique(LIBS = ['octbstack', 'm', 'oc_logger', 'connectivity_abstraction', 'coap'])
+samples_env.PrependUnique(LIBS = ['octbstack', 'm', 'connectivity_abstraction', 'coap'])
 if env.get('SECURED') == '1':
     samples_env.AppendUnique(LIBS = ['tinydtls'])
 
